diesel/src/pg/expression/functions.rs

```diff
@@ -1423,3 +1423,56 @@ define_sql_function! {
     /// ```
     fn array_upper<Arr: ArrayOrNullableArray + SingleValue>(array: Arr, dimension: Integer) -> Nullable<Integer>;
 }
+
+#[cfg(feature = "postgres_backend")]
+define_sql_function! {
+    /// Converts any SQL value to json
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// # include!("../../doctest_setup.rs");
+    /// #
+    /// # fn main() {
+    /// #     run_test().unwrap();
+    /// # }
+    /// #
+    /// # fn run_test() -> QueryResult<()> {
+    /// #     use diesel::dsl::to_json;
+    /// #     use serde_json::Value;
+    /// #     use diesel::sql_types::{Integer, Array, Json, Text, Nullable};
+    /// #     let connection = &mut establish_connection();
+    /// let result = diesel::select(to_json::<Integer, _>(1))
+    ///     .get_result::<Value>(connection)?;
+    ///
+    /// let expected: Value = serde_json::from_str("1").unwrap();
```

Comment:
can we use the `serde_json::json!` macro instead to construct the expected values?

Reply:
done